justfile

```diff
@@ -0,0 +1,31 @@
+# For documentation on `just`, see https://github.com/casey/just
```

Comment:
I just (ha!) discovered the `just` tool. It lets us write little scripts in this `justfile`, and then we can run them from a command line e.g., `just gen`, and it will spin off the proper CMake incantations so I don't have to remember them anymore. I'm an overnight fan.